dref/serializers.py

```diff
@@ -100,6 +110,15 @@ def validate(self, data):
             )
         return data
 
+    def validate_images(self, images):
+        if images:
+            for image in images:
+                if image.created_by != self.context['request'].user:
+                    raise serializers.ValidationError(ugettext('Can add images that user created'))
```

Comment:
We also need to check if the image was already attached there. For eg: if user1 owns an image and has attached the image to a DREF, then when another user2 tries to add a new image then this will throw an error.
We need to compare if the file was changed, if not then we can ignore them.

Also, let's change the text.
```suggestion
                    raise serializers.ValidationError(ugettext('Only image owner can attach image id: {image.id}'))
```